lambdas/credentialissuerstart/src/main/java/uk/gov/di/ipv/core/credentialissuer/CredentialIssuerStartHandler.java

```diff
@@ -60,6 +61,7 @@
     public static final int BAD_REQUEST = 400;
     public static final int OK = 200;
     public static final String SHARED_CLAIMS = "shared_claims";
+    public static final String ERROR_JOURNEY_STEP_URI = "/journey/error";
```

Comment:
I'm wondering it it's worth making this:
`public static final JourneyResponse errorJourneyResponse = new JourneyResponse(ERROR_JOURNEY_STEP_URI)`, and then just referencing it in the error returns below...

Reply:
Yeah good shout, I'll make the change 👍